spec/models/ticket_spec.rb

```diff
@@ -68,7 +102,7 @@
              user: user,
              ticket: ticket,
              quantity: 20)
-      expect(ticket.total_price(user)).to eq(Money.new(20 * ticket.price_cents, 'USD'))
+      expect(ticket.total_price(user)).to eq(Money.new(100000, 'USD'))
```

Comment:
Is this value supposed to be 20 (quantity in ticket purchases) * 50 (price of ticket) ?